packages/extension/gulpfile.ts

```diff
@@ -16,7 +16,15 @@ const manifest = (done) => {
   .pipe(gulp.dest('dist'))
   .on('end', () => {
     return fs.readJson('dist/manifest.json', function (err, json) {
-      json.version = pkg.version
+      let version: string = pkg.version
+
+      // https://github.com/cypress-io/cypress/issues/15032
+      // '-development' inside version string makes Chrome fail to load the extension
+      if (version.endsWith('-development')) {
+        version = version.split('-')[0]
+      }
+
+      json.version = version
```

Comment:
so... this is always `0.0.0`, right? why do we need to modify `version` here at all? it looks like it's already `0.0.0` inside of `manifest.json`:

https://github.com/cypress-io/cypress/blob/2fe5556322f8630bb0df1096f30f0bc9de284ae7/packages/extension/app/manifest.json#L40-L40

can we just... not update the version at all? delete lines 17-31?

Reply:
You're right. It's always 0.0.0. I first thought that it would be changed on every release. So, I removed the code.